src/components/ebay-menu-button/examples/19-prefix-label/template.marko

```diff
@@ -0,0 +1,19 @@
+class {
+    onCreate() {
+        this.state = { labelText: 'label text', index: -1 };
```

Comment:
Likewise, can we have the first value (i.e. Color) instead of 'label text' as the initial text.